cli/src/commands/deploy.ts

```diff
@@ -14,5 +14,7 @@ export const Deploy: CommandModule = {
       .command(DeployList)
       .command(DeployInfo)
       .command(DeployLogs),
-  handler: DeployCreate.handler,
```

Comment:
Having `differential deploy` default to create could be unsafe (as there are sub-commands on `deploy` i.e `list`, `info`.